corehq/apps/case_search/filter_dsl.py

```diff
@@ -251,6 +251,8 @@ def build_filter_from_xpath(domain, xpath, fuzzy=False):
         "Please try reformatting your query. "
         "The operators we accept are: {}"
     )
+    if isinstance(xpath, list):
+        xpath = ' and '.join(xpath)
```

Comment:
rather than handling this here I think we should update `CaseSearchQueryBuilder` to add each expression to the query individually:

https://github.com/dimagi/commcare-hq/blob/5db3d308ab57400e496999e276c40afd8e23e97b/corehq/apps/case_search/utils.py#L170-L172

You can use `criteria.has_multiple_terms` to determine if it is a list or not.